ampd/src/grpc/mod.rs

```diff
@@ -35,6 +36,8 @@ pub struct Config {
     pub port: u16,
     pub concurrency_limit: nonempty::Usize,
     pub concurrency_limit_per_connection: nonempty::Usize,
```

Comment:
what's the difference between the two?

Reply:
One is the global limit of concurrent requests, and the other is the limit per HTTP2 connection.